test/common/router/rds_impl_test.cc

```diff
@@ -483,47 +490,33 @@ TEST_F(RouteConfigProviderManagerImplTest, Basic) {
   EXPECT_EQ(0UL, configured_providers.size());
 }
 
-TEST_F(RouteConfigProviderManagerImplTest, onConfigUpdateEmpty) {
-  std::string config_json = R"EOF(
-    {
-      "cluster": "foo_cluster",
-      "route_config_name": "foo_route_config",
-      "refresh_delay_ms": 1000
-    }
-    )EOF";
-
-  Json::ObjectSharedPtr config = Json::Factory::loadFromString(config_json);
-  envoy::api::v2::filter::network::Rds rds;
-  Envoy::Config::Utility::translateRdsConfig(*config, rds);
+// Negative test for protoc-gen-validate constraints.
```

Comment:
Worth leaving this code for now? Or just delete and add TODO up above? I'm fine either way. Maybe also link to issue tracking this?

Reply:
Coverage is broken until some recent PGV fixups make their way across to `data-plane-api`. Need to merge https://github.com/lyft/protoc-gen-validate/pull/40 first though. So, will wait until that all happens, then this test should work for real.